docs/reference/transform/painless-examples.asciidoc

```diff
@@ -573,11 +575,11 @@ POST _transform/_preview
                 all_docs.add(span); 
               }
             }
-            all_docs.sort((HashMap o1, HashMap o2)->o1['@timestamp'].millis.compareTo(o2['@timestamp'].millis)); 
+            all_docs.sort((HashMap o1, HashMap o2)->o1['@timestamp'].compareTo(o2['@timestamp'])); 
             def size = all_docs.size();
             def min_time = all_docs[0]['@timestamp'];
             def max_time = all_docs[size-1]['@timestamp'];
-            def duration = max_time.millis - min_time.millis;
+            def duration = max_time.second - min_time.second;
```

Comment:
Yes, millis is not available anymore.

Reply:
@hendrikmuhs I updated the response via 6710dea

Reply:
I will investigate why `milis` has vanished. I think this can break user upgrades for no good reason.

I also think that the example is less useful with seconds, measuring in millis is quite common.

Reply:
@hendrikmuhs Do you think it's still worth applying the changes in this PR to have the example working for the time being?

Reply:
it seems `toEpochMilli()` is the new `millis`

Reply:
Awesome, thanks! I tested it and change the script and the response in f7baa0c.